Changed title to Trainer Training 2020 (#612)
carpentries/carpentries.org: content/support/membership/_index.md

---
title: The Carpentries Partnerships
layout: single
cascade:
  nav: partnership
widgets:
- newsletter
aliases:
- /membership/
- /partnership/
---

## Become A Partner

Jump start your local data skills training efforts with a Carpentries Partnership. 
By partnering with The Carpentries, organisations join a global, professional skill-building community dedicated to teaching foundational data and computational skills. 
Partners gain access to a comprehensive suite of training programs, consultation services, community engagement opportunities, and resources, enabling them to grow local communities focused on teaching and learning data skills.

> ## Why Partnership?

Reviewer: ```suggestion
## Why a Partnership?
```

   </table>
{{< /table >}}

## Partnership Levels

Our Partnership program is flexible to enable you to build the service package that works best for your team. 
We offer four Partnership levels and encourage you to contact us to discuss more customisation options. 

Each Partnership level comes with a set number of service options that can be mixed and matched from the following: 

> - one (1) workshop using one of our data science or computational skills curricula, or

Reviewer: Could we rather use lessons than Curricula? Might be more inclusive

Reviewer 2: Good idea @elletjies. We want to avoid confusion between a "lesson" which is a single topic and a "curriculum" which is a series of lessons, but I agree that "curricula" is not as commonly used and could be confusing. I'm changing to "courses".

> - one (1) workshop using one of our data science or computational skills curricula, or

Reviewer 2: ```suggestion
- one (1) workshop using one of our data science or computational skills courses, or
```

- two (2) seats in a certification program (Instructor Training or Collaborative Lesson Development Training), or
- four (4) seats in our Learner-Centered Teaching program, or
- sixteen (16) seats in one of our short-format professional development courses, or
- any fractional combination of the above.

> Our **Launch** level subscription allows you to choose one of the above service options. 

Reviewer: ```suggestion
Our **Launch** level subscription allows you to choose one (1) of the above service options. 
```

> Partnership packages at the **Expand**, **Scale**, and **Transform** levels allow you to select three, five, and eight service options from the list above, respectively. Additional services can be added on to your Partnership at any time during the Partnership term. 

Reviewer: ```suggestion
Partnership packages at the **Expand**, **Scale**, and **Transform** levels allow you to select three (3), five (5), and eight (8) service options from the list above, respectively. Additional services can be added on to your Partnership at any time during the Partnership term. 
```

> [View our current pricing tiers](/support/pricing/#partnership-pricing).

Reviewer: See my notes about levels and tiers above

> [View our current pricing tiers](/support/pricing/#partnership-pricing).

Reviewer 2: ```suggestion
[View our current pricing](/support/pricing/#partnership-pricing).
```

Reviewer 2: Here the language can be simplified to just "view our current pricing"

{.button} 

### Launch Partnership

Appropriate for organisations in the early stages of establishing a local community of practice around data science and computational research skills, a Launch Partnership can be used to demonstrate the uniqueness and impact of The Carpentries teaching model, as a stepping-stone towards securing sustained organisational investment. 

### Expand Partnership

For organisations ready to increase their training capacity, an Expand Partnership provides multiple options for skill development and community building. 

> Organisations at the Expand level may elect to train up to six individuals as certified Carpentries Instructors or Lesson Developers, spread evidence-based teaching practices more broadly across their departments with up to twelve seats in our Learner-Centered Teaching program, or combine either of these paths with short-format professional development opportunities for their staff and students.

Reviewer: ```suggestion
Organisations at the Expand level may elect to train up to six (6) individuals as certified Carpentries Instructors or Lesson Developers, spread evidence-based teaching practices more broadly across their departments with up to twelve (12) seats in our Learner-Centered Teaching program, or combine either of these paths with short-format professional development opportunities for their staff and students.
```

### Scale Partnership

Designed for organisations seeking to broaden their impact, a Scale Partnership enables expanded programming for long-lasting growth of overall institutional capacity. 

> Partners at the Scale level can embed The Carpentries philosophy of teaching and learning broadly throughout their organisation with up to twenty seats in our Learner-Centered Teaching program, or choose to concentrate on certification of up to ten individuals as Carpentries Instructors or Lesson Developers. 

Reviewer: ```suggestion
Partners at the Scale level can embed The Carpentries philosophy of teaching and learning broadly throughout their organisation with up to twenty (20) seats in our Learner-Centered Teaching program, or choose to concentrate on certification of up to ten individuals as Carpentries Instructors or Lesson Developers. 
```

They can also reward existing community members with additional professional development opportunities. 

### Transform Partnership

> Intended for organisations pursuing truly institution-wide change, a Transform Partnership enables accelerated large-scale capacity growth and sustained, organisation-wide impact. 

Reviewer 3: ```suggestion
Intended for organisations pursuing even broader institution-wide change, a Transform Partnership enables accelerated large-scale capacity growth and sustained, organisation-wide impact. 
```

> Transform level Partners can certify up to sixteen individuals as Carpentries Lesson Developers or Instructors, providing the person-power for extensive curriculum development and instructional efforts. 

Reviewer: ```suggestion
Transform level Partners can certify up to sixteen (16) individuals as Carpentries Lesson Developers or Instructors, providing the person-power for extensive curriculum development and instructional efforts. 
```

* Personalised consultation with a member of The Carpentries team to develop a customised training package that meets your organisation's goals.
* Access to resources and support for establishing and maintaining a thriving local community around teaching and learning data science and computational skills.
* Opportunities to connect with and contribute to our global community through:
  * Sharing job postings, research projects, and training initiatives with our worldwide network.
  * Participating in community calls, forums, and collaborative projects.
* Quarterly reports on service usage and trainee progress towards certification.
* Opportunity to submit a candidate for our Instructor Trainer Training course, through which individuals can become certified to teach our Instructor Training courses (additional fees apply).
* Unlimited self-organised workshops.
* Recognition of the valuable service contributions made by your organisational members to the global Carpentries community.
* Access to discounted pricing for additional services purchased during the Partnership year.

[Become a Partner]({{< param membership_form >}})
{.button} 

## Contact Us

> Have Questions? Please consult our [Partnership FAQ](/support/membership/partnership-faq/) or contact our team at [{{< param membership_email >}}](mailto:{{< param membership_email >}}).

Reviewer 3: ```suggestion
Have Questions? Please consult our [Partnership FAQ](/support/partnership/partnership-faq/) or contact our team at [{{< param membership_email >}}](mailto:{{< param membership_email >}}).
```

> Have Questions? Please consult our [Partnership FAQ](/support/membership/partnership-faq/) or contact our team at [{{< param membership_email >}}](mailto:{{< param membership_email >}}).

Reviewer 3: Also for @maneesha - checking if this should not be to the partnership email instead.
